release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2732,6 +2732,54 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.15 --pullspecs
+----
+
+[id="ocp-4-15-15-bug-fixes"]
+==== Bug fixes
+
+* Previously, certain situations caused transfer of an Egress IP address from one node to a different node to fail, and this failure impacted the OVN-Kubernetes network. The network failed to send gratuitous Address Resolution Protocol (ARP) requests to peers to inform them of the new node's medium access control (MAC) address. As a reult, peers would temporarily send reply traffic to an old node and this traffic led to failover issues. With this release, the OVN-kubernetes network correctly sends a gratuitous ARP to peers to inform them of the new Egress IP node MAC address, so that each peer can send reply traffic to the new node without causing failover time issues. (link:https://issues.redhat.com/browse/OCPBUGS-33960[*OCPBUGS-33960*]
+
+* Previously, when mirroring operator catalogs, the `oc-mirror` rebuilt the catalogs and regenerated the catalogs internal cache according to the `imagesetconfig` catalog filtering specifications. This operation required the use of the `opm` binary found within the catalogs. In {product-title} 4.15, operator catalogs include the `opm` {op-system-base-full} 9 binary and this caused the mirroring process to fail when running on {op-system-base} 8 systems. With this release, `oc-mirror` no longer builds catalogs by default. Instead, catalogs are mirrored directly to their destination registries. (link:https://issues.redhat.com/browse/OCPBUGS-33575[*OCPBUGS-33575*]
+
+* Previously, the image registry did not support Amazon Web Services (AWS) region `ca-west-1`. With this release, the image registry can now be deployed in this region. (link:https://issues.redhat.com/browse/OCPBUGS-33672[*OCPBUGS-33672*])
+
+* Previously, ServiceAccounts (SAs) could not be used as OAuth2 clients because there were no tokens associated with the SAs. With this release, the OAuth registry client has been modified to anticipate this case and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33210[*OCPBUGS-33210*])
+
+* Previously, the proxy information set in the install config was not applied to the bootstrap process. With this release, the proxy information is applied to the bootstrap ignition data which is applied to the bootstrap machine and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33205[*OCPBUGS-33205*])
```

Comment:
Capitalize "Ignition":
```suggestion
* Previously, the proxy information set in the install config was not applied to the bootstrap process. With this release, the proxy information is applied to the bootstrap Ignition data which is applied to the bootstrap machine and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33205[*OCPBUGS-33205*])
```
I think also (from looking at the original, pre-clone [Jira](https://issues.redhat.com/browse/OCPBUGS-12890)) that this would more accurately (and align with how the rest of the repo refers to it) be:
```suggestion
* Previously, the proxy information set in the `install-config.yaml` file was not applied to the bootstrap process. With this release, the proxy information is applied to the bootstrap Ignition data which is applied to the bootstrap machine and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33205[*OCPBUGS-33205*])
```
Or if you'd rather be a little more generic just to be safe, at least add "file":
```suggestion
* Previously, the proxy information set in the install config file was not applied to the bootstrap process. With this release, the proxy information is applied to the bootstrap Ignition data which is applied to the bootstrap machine and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33205[*OCPBUGS-33205*])
```